src/System.Windows.Forms/tests/InteropTests/System.Windows.Forms.Interop.Tests.csproj

```
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>System.Windows.Forms.Interop.Tests</AssemblyName>
```

To avoid namespace clashes
```suggestion
    <AssemblyName>System.Windows.Forms.Interop.Tests</AssemblyName>
    <RootNamespace>System.Windows.Forms.InteropTests</RootNamespace>
```

---

Assembly name needs to end with `.Tests` to be recognised by the arcade infrastructure as a test project unfortunately, so this is why I did this deliberately